tests/system/providers/google/cloud/cloud_build/example_cloud_build_trigger.py

```diff
@@ -124,12 +126,16 @@
         task_id="delete_build_trigger",
         project_id=PROJECT_ID,
         trigger_id=build_trigger_id,
+        trigger_rule=TriggerRule.ALL_DONE,
```

Comment:
Move trigger_rule outside START/END documentation marks

Reply:
Done
Thanks for the review!